linera-faucet/server/src/lib.rs

```diff
@@ -145,46 +145,48 @@ where
                 // tokens remain locked, so the remaining balance must be at least 1/3 of the start
                 // balance. In general:
                 // start_balance / full_duration <= remaining_balance / remaining_duration.
-                if Self::multiply(u128::from(self.start_balance), remaining_duration)
-                    > Self::multiply(u128::from(remaining_balance), full_duration)
+                if multiply(u128::from(self.start_balance), remaining_duration)
+                    > multiply(u128::from(remaining_balance), full_duration)
                 {
                     return Err(Error::new("Not enough unlocked balance; try again later."));
                 }
             }
         }
 
-        let ownership = ChainOwnership::single(owner);
         let result = client
-            .open_chain(ownership, ApplicationPermissions::default(), self.amount)
+            .open_chain(
+                ChainOwnership::single(owner),
+                ApplicationPermissions::default(),
+                self.amount,
+            )
             .await;
         self.context.lock().await.update_wallet(&client).await?;
-        let (chain_id, certificate) = match result? {
-            ClientOutcome::Committed(result) => result,
+        let chain_id = match result? {
+            ClientOutcome::Committed((chain_id, _certificate)) => chain_id,
             ClientOutcome::WaitForTimeout(timeout) => {
                 return Err(Error::new(format!(
                     "This faucet is using a multi-owner chain and is not the leader right now. \
-                    Try again at {}",
+                      Try again at {}",
```

Comment:
(I always wonder about that; is that the correct way to indent multi-line string literals? :sweat_smile:)

Reply:
Oh this was just some combination of my editor and `cargo fmt` :sweat_smile: Reverting.